clients/docs/java-springboot.rst

```diff
@@ -0,0 +1,176 @@
+.. _client-examples-java-springboot:
+
+Java Spring Boot
```

Comment:
The java springboot page isn't rendering quite right.  Let's enable you to build locally so you can generate the docs on your laptop tomorrow.